commons/src/main/java/com/connexta/ion/replication/spring/ReplicationProperties.java

```diff
@@ -64,4 +70,24 @@ public int getReceiveTimeout() {
   public void setReceiveTimeout(int receiveTimeout) {
     this.receiveTimeout = receiveTimeout;
   }
+
+  /**
+   * Gets the internal set of all destination site identifiers handled by this replicator or empty
+   * to have all sites handled.
```

Comment:
✏️ What do you mean by the internal set? Could you just say "Gets the set of all destination site identifiers handled by this replicator..."

Reply:
I mean it returns the actual set of identifiers managed internally. Although I hate this (and Pojos), I have to return a reference to the internal set so that Spring will be able to populate it.